examples/guides/animation/step-6.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <!-- Include CARTO VL JS -->
+    <script src="../../../../dist/carto-vl.js"></script>
+    <!-- Include Mapbox GL JS -->
+    <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
+    <!-- Include Mapbox GL CSS -->
+    <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
+    <!-- Include custom CARTO styles -->
+    <link rel="stylesheet" type="text/css" href="../../style.css">
+</head>
+
+<body>
+    <!-- Map goes here -->
+    <div id="map"></div>
+    <script>
+        // Add basemap and set properties
+        const map = new mapboxgl.Map({
+            container: 'map',
+            style: carto.basemaps.darkmatter,
+            center: [-22.6995, 41.94674],
+            zoom: 2.8,
+            scrollZoom: false
+        });
+        map.touchZoomRotate.disableRotation();
```

Comment:
Now that we have tilt and rotation, we could remove this from these examples, so animations can be displayed also in 2.5D